datahub-web-react/src/app/search/SearchPage.tsx

```diff
@@ -62,6 +62,7 @@ export const SearchPage = () => {
                 searchFlags: { getSuggestions: true },
             },
         },
+        fetchPolicy: 'cache-and-network',
```

Comment:
sweet so this will show cached results first and then once the network request returns it will update that cache and show updated results?

Reply:
yes! So if you change tags, owners, etc and return to search page we will indeed get the update for the user